### PR TITLE
Improving unit test code coverage for PagerProviderRegistry

### PR DESCRIPTION
**Pull request by jandom**

### CODE REVIEW

File: tests/Unit/Provider/PagerProviderRegistryTest.php [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+ * This file is part of the FOSElasticaBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+/**
+ * This file is part of the FOSElasticaBundle project.
```
Comment (XWB) [inline tests/Unit/Provider/PagerProviderRegistryTest.php]: Duplicate header comments can be removed.

Comment (jandom) [reply tests/Unit/Provider/PagerProviderRegistryTest.php]: A very good point sir!

Comment (jandom) [reply tests/Unit/Provider/PagerProviderRegistryTest.php]: Done now

**Comment by XWB** [comment]
Thank you once again :)

**Pull request merged**